test.html

```diff
@@ -0,0 +1,34 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="test.css" />
+    <title>Document</title>
+  </head>
+  <body>
+    <div>
+        <p class="score">0    :    0</p>
+    </div>
+    <p class="result">시작하려면 가위, 바위, 보 중 하나를 선택하세요</p>
+      <div class="block">
+        <h3>인간</h3>
+        <h3>컴퓨터</h3>
+      </div>
+      <div class="block">
+        <p class="yourChoice">🪨</p>
+        <p class="computerChoice">🪨</p>
+      </div>

```

Comment:
✨P5: 블록을 나누는 기준을 세로(사람/컴퓨터)로 하는 것이 어떨까요? 구현해야 하는 그림을 살펴보면, 사람과 컴퓨터 블록은 스타일도, 적용해야하는 기능도 비슷합니다. 블록을 가로 기준이 아닌 세로 기준으로 나누는 것도 좋을 것 같아요!
```suggestion
      <div class="result-container">
        <div class="human">
          <p class="name">인간</p>
          <p class="user-choice"></p>
        </div>
        <div class="computer">
          <p class="name">컴퓨터</p>
          <p class="computer-choice"></p>
        </div>
      </div>    
```
이런 식으로 사람과 컴퓨터로 분리할 수 있습니다. 이렇게 하면 human 클래스와 computer 클래스의 css 값은 아래와 같이 한 번에 참조할 수 있어 편리할 것 같아요!
```
.human,
.computer {
  display: flex;
  flex-direction: column;
  justify-content: center;
  align-items: center;
  .name {
    font-weight: 700;
    font-size: 20px;
  }
}
```